include/fiction/algorithms/simulation/sidb/time_to_solution.hpp

```diff
@@ -82,7 +85,8 @@ void sim_acc_tts(const Lyt& lyt, const quicksim_params& quicksim_params, time_to
     const auto simulation_results_exgs = exhaustive_ground_state_simulation(lyt, quicksim_params.phys_params);
 
     time_to_solution_stats st{};
-    st.single_runtime_exhaustive = mockturtle::to_seconds(simulation_results_exgs.simulation_runtime);
+    st.single_runtime_exhaustive =
+        units::time::second_t(mockturtle::to_seconds(simulation_results_exgs.simulation_runtime));
```

Comment:
Can you use braced initialization here, please? It is generally preferrable to avoid accidentally declaring a function.